mlir/include/mlir/Dialect/AMDGPU/IR/AMDGPU.td

```diff
@@ -765,4 +771,37 @@ def AMDGPU_WMMAOp :
   let hasVerifier = 1;
 }
 
+def GlobalLoadMemRefType : MemRefOf<[GlobalLoadTypes]>;
+
+def AMDGPU_GlobalLoadLDSOp :
```

Comment:
1. This doesn't need "global" in the name - it'll work for buffer intrinsic as well
2. This operation has some rather strong and nonobvious gather semantics - could it be `amdgpu.gather_to_lds` instead? Or `amdgpu.direct_lds_gather`? ...

Reply:
I changed to `amdgpu.gather_to_lds`